source/sgp_mode/CPU.h

```diff
@@ -77,7 +87,8 @@ class CPU {
    * Purpose: Mutates the genome code stored in the CPU.
    */
   void Mutate() {
-    program.ApplyPointMutations(0.03);
+    program.ApplyPointMutations(state.world->GetConfig()->MUTATION_SIZE() *
```

Comment:
This should probably be attached to MUTATION_RATE instead of size, since size doesn't really make sense when we're talking about instructions instead of numbers

Reply:
Yeah, I was thinking about that but the main reason I used MUTATION_SIZE is because the default value is pretty reasonable, whereas MUTATION_RATE defaults to 1; also, the description of MUTATION_RATE doesn't really fit because it describes it as the chance that any mutation occurs at all, whereas this is more like how much to mutate and thus closer to MUTATION_SIZE.